core/src/main/java/greencity/controller/EcoNewsController.java

```diff
@@ -369,6 +369,7 @@ public ResponseEntity<Integer> countLikesForEcoNews(@PathVariable Long econewsId
      */
     @Operation(summary = "Check if user liked news")
     @GetMapping("/isLikedByUser")
+    @ApiResponse(responseCode = "401", description = HttpStatuses.UNAUTHORIZED)
     public ResponseEntity<Boolean> checkNewsIsLikedByUser(@RequestParam("econewsId") Long econewsId,
                                                           @Parameter(hidden = true) @CurrentUser UserVO user) {
         return ResponseEntity.status(HttpStatus.OK).body(ecoNewsService.checkNewsIsLikedByUser(econewsId, user));
```

Comment:
The 401 response status documentation should`ve been added to getEcoNewsByUserByPage method but was added to the checkNewsIsLikedByUser method instead